app/2.0.x/pdk/kong.ctx.md

```diff
@@ -19,9 +25,9 @@ A table that has the lifetime of the current request and is shared between
  Since only relevant in the context of a request, this table cannot be
  accessed from the top-level chunk of Lua modules. Instead, it can only be
  accessed in request phases, which are represented by the `rewrite`,
- `access`, `header_filter`, `body_filter`, and `log` phases of the plugin
- interfaces.  Accessing this table in those functions (and their callees) is
- fine.
+ `access`, `header_filter`, `body_filter`, `log`, and `preread` phases of
```

Comment:
The `preread` phase was missing on several places on this file. It exists on the 2.0.4 docs, so it was added here. I ignore why it wasn't there previously.